lib/commons/aria/get-owner-virtual.js

```diff
@@ -0,0 +1,54 @@
+/* global aria, dom */
+
+/**
+ * Get owner element of a provided virtualNode
+ * @param {Object} virtualNode virtual node
+ * @returns {HTMLElement | undefined}
```

Comment:
This needs to return virtual nodes. We have to make sure this works on SerialVirtualNode too.